### DISCUSSION TITLE
extension fails to open a notebook after workspace restart

### DISCUSSION THREAD

**Discussion by Ratgor** [microsoft/vscode-jupyter#10547] [Questions and Answers] [answered]
Extensions:
Jupyter (ms-toolsai) v2022.5.1001610507
Jupyter Keymap (ms-toolsai) v1.0.0
Jupyter Notebook Renderers (ms-toolsai) v1.0.8
Python (ms-python) v2022.8.0

Running at
Eclipse Che VS Code Version: 1.68.0
Chrome  102.0.5005.115 
Windows 10

Opens jupyter notebooks successfully at the first run,
but after workspace restart (not re-creation)
any notebook opening fails:

"Unable to open 'filename.ipynb' Cannot open resource with notebook editor type 'jupyter-notebook', please check if you have the right extension installed and enabled." 
(of course, it is installed, enabled, and reinstall doesn't help, earlier versions too) 
+
"Activating extension 'vscode.ipynb' failed: o.notebooks.onDidChangeNotebookCells is not a function."

Is it a bug? of the extension or vscode or a complex system?
should it be reported here? (there are some similar, but closed)

**Comment by IanMatthewHuff** [comment, marked as answer]
@Ratgor I believe that you would have to take this up / file an issue with Eclipse Che on this. I'm not fully sure how Che updates, but that error looks like they don't have the updated notebook API in the core of VS Code. And from our side we can only control our extension and the main VS Code release we can't fix anything with Che.
